### PR TITLE
Bump to 0.3.0.

### PR DESCRIPTION
**Pull request by jmjoy**

### CODE REVIEW

File: e2e/Cargo.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #
 [package]
 name = "e2e"
-version = "0.2.0"
+version = "0.0.0"
```
Comment (jmjoy) [inline e2e/Cargo.toml:20]: Change to `0.0.0` because the version of e2e isn't important.

Comment (wu-sheng) [reply e2e/Cargo.toml:20]: 0.0.0 means using the latest version?

**Comment by jmjoy** [comment]
There is a question, does e2e source code have to be included to `.crate`? Because e2e is integrated into worksapce, there is no e2e source code when run `cargo package`.

**Comment by wu-sheng** [comment]
Once users are not being affected, then it is not an issue. You could choose a way which is more convenience for developers.

File: e2e/Cargo.toml (hunk 1 above)
Comment (jmjoy) [reply e2e/Cargo.toml:20]: Should mean there is no version, since it doesn't have to be published to `crates.io`, it doesn't matter.

**Comment by jmjoy** [comment]
> Once users are not being affected, then it is not an issue. You could choose a way which is more convenience for developers.

Then it's fine.

File: e2e/Cargo.toml (hunk 1 above)
Comment (wu-sheng) [reply e2e/Cargo.toml:20]: I mean does this guarantee it would use the latest build version?

Comment (jmjoy) [reply e2e/Cargo.toml:20]: This line insures to use the latest version.
<https://github.com/apache/skywalking-rust/blob/d14a0bd4c781671e907c3cb796f20ef23d6ed114/e2e/Cargo.toml#L27>

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**